core/templates/dev/head/library/Library.js

```diff
@@ -79,7 +79,8 @@ oppia.controller('Library', [
 
       var windowWidth = $(window).width() * 0.85;
       $scope.tileDisplayCount = Math.min(
-        Math.floor(windowWidth / tileDisplayWidth), MAX_NUM_TILES_PER_ROW);
+        Math.floor(windowWidth / (tileDisplayWidth + 20)),
```

Comment:
@seanlip I'll add the comment. The reasoning for the number 20 is same as @kevinlee12 explained.